src/cromshell/utilities/http_utils.py

```diff
@@ -61,3 +63,29 @@ def check_http_request_status_code(
                 f"Status_code: {response.status_code}\n"
                 f"Message: {response.text}"
             )
+
+
+def generate_headers(config):
```

Comment:
No change needed, just calling out the behavior that presumably this runs on every request, rather than once-per-cromshell. I assume that could be a good thing (tokens might change) OR a bad thing (if commands run lots and lots of commands right after each other).

Reply:
I was thinking more along the lines of keeping things functional and not creating state if it can be avoided. As a bonus not putting a token in memory makes it less likely to end up somewhere it shouldn't.

It's true that we may call `print-access-token` 2 or 3 times during a single invocation, but each call takes only about 1.1 seconds on my machine.